### DISCUSSION TITLE
Make blood come out from the neck after you headshot a enemy.

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1183] [Ideas and Feature Requests]
Something i've been thinking..
When you headshot a enemy with a rifle, and rips their head off, there's barely any blood that comes out.
What if the blood particles just kept coming off from the neck bone for that, until the player/enemy respawns?

**Comment by ghost** [comment]
Is it possible to make particle effects more realistic overall?

**Comment by Calinou** [comment]
@Rafirzm You need to be more specific so we can look at implementing such features :slightly_smiling_face:

The engine supports basic particle collision (which makes particles disappear when hitting a surface), but there no advanced physics like bounding particles.

**Comment by ghost** [comment]
@Calinou Magicolo Game 2014, they had mastered it well, won't harm Halo but they had nice effects with weapons in the fourth.

**Comment by ghost** [comment]
I think it will be fine for it to be solved meanwhile this issue with particle effects doesn't need to be discussed since it doesn't concern the game in general but the particle effect integration into this game. Still, some games greatly build with Tesseract benefits a tremendeous gestion of particules as well.

**Comment by q009** [comment]
Particle overhaul already on my TODO list.
